hs628_tools/Test.py

```diff
@@ -0,0 +1,133 @@
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+
+
+
+
+
+class data_split:
+    """
+    Create class data_split to predict a perfect train and test data split ratio.
+
+    defines method to accomplish this task.
+
+    returns a split ratio.
+
+    Identifies Missing values as Nan for any missing value, return train and test dataframe with uniform division
+    of NAN between train and test data per the data split ratio.
+
+    """
+    def __init__(self, data):
+        self.data = data
+
+
+    def test_train():
+        """
+        Reads in csv file using np.genfromtxt, counts number of NANs in the dataset
+
+        """
+
+        data = np.genfromtxt(csv_datafile, delimiter=',', skip_header=1)
+        np.isnan(data).sum() #counts the number of "NANs" in dat
+        #print data.shape
+
+
+
+        #set calculation standard for specific datalengths, this is based on the general rule of thumb
+        #ratio increases as the length of the data increases, so to ensure we have  enough data to train and test
+        #for the smaller datasets
+        
+        if len(data) <= 500:
+            train = 60
+            test = 40
+        elif len(data) > 500 and row_len <= 10000:
+            train = 70
+            test = 30
+        elif len(data) > 10000 and row_len <=100000:
+            train = 75
+            test = 25
+        else:
+            train = 80
+            test = 20
+        print 'train:',train, 'test:',test
+
+
+        # drops all NAN values from data
+        non_nandata = data[~np.isnan(data).any(axis=1)]
+
+        #print non_nandata.shape
+
+        #print non_nandata.shape
+
+
+
+        ### nan dataframe
```

Comment:
You are not really making a dataframe here, and the array you make is not 100% NaN's, the comment should be more clear.

Reply:
Modified comment per the recommendation to make it more clear to the users.